docs/README.md

```diff
@@ -5,7 +5,7 @@ They are published on powsybl.readthedocs.io/ and pull requests are built and pr
 
 To build the docs locally, run the following commands:
 ~~~bash
-pip install -r docs/requirements.txt
-sphinx-build -a docs ./build-docs
+pip install -r requirements.txt
```

Comment:
```suggestion
pip install -r docs/requirements.txt
```
Leave it as it was!

Reply:
Why? The README is located in repositiory docs/.

Reply:
![image](https://github.com/powsybl/powsybl-core/assets/48123713/490dd012-a758-494c-b88a-8cef8fb220c1)